docs/readme/README.tr.md

```
@@ -0,0 +1,295 @@
+<img src="../../assets/rename-banner.svg" alt="Duyuru: AltSendme artık DashBeam oldu. Aynı uygulama; bulması, hatırlaması ve telafuz etmesi daha kolay." width="1200" />
```

markdownlint-cli2 (0.23.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)